server/i18n/all.pot

```diff
@@ -874,663 +874,3 @@ msgstr "Nitrous oxide emissions (large facilities, non-biogenic)"
 #. Title of a place chart: Water withdrawal rate
 msgid "CHART_TITLE-water_withdrawal_rate"
 msgstr "Water withdrawal rate"
-
```

Comment:
hm, curious why these got deleted, @beets?

Reply:
yes - these should not be deleted. @raebo1017 for now, could you revert all the i18n files?